hummingbot/templates/conf_global_TEMPLATE.yml

```diff
@@ -45,6 +45,7 @@ celo_password: null
 # Ethereum wallet address: required for trading on a DEX
 ethereum_wallet: null
 ethereum_rpc_url: null
+ethereum_rpc_ws_url: null
```

Comment:
We need to increase the template_version by one every time we add a new param.

Reply:
What I mean is you need to update `template_version` (line 6 of this file) to 9.